lib/model/QubitInformationObject.php

```diff
@@ -672,18 +672,17 @@ protected function updateNestedSet($connection = null)
   public function getDescendantsForExport($options = array())
   {
     $descendants = array();
-    $fromClipboard = isset($options['params']['fromClipboard']) ? true : false;
     $levels = isset($options['levels']) ? $options['levels'] : array();
     $numLevels = count($levels);
 
     foreach ($this->getChildren()->orderBy('lft') as $child)
     {
       $addCondition = true;
 
-      // If we're not in a CLI or BG job enviroment, check ACL
-      if (!is_using_cli() || $fromClipboard)
+      // If we're not in a CLI enviroment, check ACL
+      if ('cli' != sfContext::getInstance()->getConfiguration()->getEnvironment())
       {
-        $addCondition = QubitAcl::check($child, 'read');
+        $addCondition = QubitAcl::check($child, 'read', $options);
```

Comment:
What exactly is in `$options` that you need to pass to `QubitAcl`? `levels`? `public`?